File: backend-jakarta/src/main/java/com/example/backendjakarta/models/OrderContentModel.java
package com.example.backendjakarta.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OrderContentModel {
    private int orderId;
    private MenuItemModel menuItem;
    private int quantity;
}

File: backend-jakarta-ee/src/main/kotlin/com/example/backendjakartaee/models/OrderContentModel.kt
package com.example.backendjakartaee.models

import jakarta.json.bind.annotation.JsonbCreator
import jakarta.json.bind.annotation.JsonbProperty

data class OrderContentCreateModel @JsonbCreator constructor(
    @JsonbProperty("menuItem") var menuItemId: Int,
    @JsonbProperty("quantity") var quantity: Int
)

data class OrderContentModel @JsonbCreator constructor(
    @JsonbProperty("orderId") var orderId: Int,
    @JsonbProperty("menuItem") var menuItemId: Int,
    @JsonbProperty("quantity") var quantity: Int
)